guides/video_pagination.md

```diff
@@ -0,0 +1,138 @@
+# Video Pagination with Active Speaker-Based Policy
+
+## Overview
+
+Amazon Chime SDK currently supports up to 16 concurrent video streams in a meeting. However, in many cases, the application may not want to render all the available video streams, because of the following reasons:
```

Comment:
Do we want to omit the number 16 and reference the AWS doc incase this changes in the future? https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits